style/ruby/README.md

```diff
@@ -3,41 +3,23 @@ Ruby
 
 [Sample](sample.rb)
 
-* Avoid conditional modifiers (lines that end with conditionals).
 * Avoid multiple assignments per line (`one, two = 1, 2`).
-* Avoid organizational comments (`# Validations`).
 * Avoid ternary operators (`boolean ? true : false`). Use multi-line `if`
   instead to emphasize code branches.
 * Avoid explicit return statements.
 * Avoid using semicolons.
-* Avoid bang (!) method names. Prefer descriptive names.
```

Comment:
Bang means it is an action that will change the state of the object. Don't see why not take advantage of the semantics behind that. With that said, no es algo que hemos usado mucho.